base/sysimg.jl

```diff
@@ -548,6 +549,12 @@ Base.require(:Unicode)
 
     # PR #25571
     @deprecate_binding LinAlg root_module(:LinearAlgebra) true ", run `using LinearAlgebra` instead"
+
+    # PR 25544
+    @deprecate_binding REPL            root_module(:REPL)                 true ", run `using REPL` instead"
+    @deprecate_binding LineEdit        root_module(:REPL).LineEdit        true ", use `REPL.LineEdit` instead"
+    @deprecate_binding REPLCompletions root_module(:REPL).REPLCompletions true ", run `REPL.REPLCompletions` instead"
```

Comment:
If `REPLCompletions` becomes a submodule of `REPL`, what about renaming it to `REPL.Completions`?  `REPL.REPLCompletions` looks redundant.